tests/python/gaia-ui-tests/gaiatest/apps/gallery/app.py

```diff
@@ -73,12 +73,15 @@ def empty_gallery_text(self):
 
     @property
     def are_gallery_items_displayed(self):
-        return self.marionette.find_element(*self._gallery_items_locator).is_displayed()
+        return self.is_element_displayed(*self._gallery_items_locator)
 
     @property
     def thumbnails(self):
-        return [self.Thumbnail(self.marionette, thumbnail, index)
```

Comment:
This will go faster in case there are no gallery items. Normally, find_elements will keep on looking for 10s and then will return an empty list, see: http://mxr.mozilla.org/mozilla-central/source/testing/marionette/driver/marionette_driver/marionette.py#1583